apps/examples/iotjs_startup/Kconfig

```diff
@@ -39,12 +39,14 @@ if EXAMPLES_IOTJS_STARTUP_WIFI
 config EXAMPLES_IOTJS_STARTUP_WIFI_SSID
 	string "SSID of AP"
 	default "public"
+	depends on EXAMPLES_IOTJS_STARTUP_WIFI
```

Comment:
I am sorry but last comment. 
1. If you intend to use ```depends on EXAMPLES_IOTJS_STARTUP_WIFI```, let's remove line 37. ```if EXAMPLES_IOTJS_STARTUP_WIFI``` and line 52, ```endif```. That is duplicated with depends on.
2. If you want to keep ```if xx``` instead of ```depends on xx```, you should move endif location to cover EXAMPLES_IOTJS_STARTUP_WIFI_AUTH and EXAMPLES_IOTJS_STARTUP_WIFI_CRYPTO.